### PR TITLE
Updated the runner to netstandard1.5 and added wait-command parameter

### PR DESCRIPTION
**Pull request by livarcocc**
Updating the runner to netstandard1.5. Also, modifying the runner to deal with the wait-command parameter, so that we can send tests inside the a message as opposed through the command line, which has a length limit.

cc @piotrpMSFT

### CODE REVIEW

File: src/dotnet-test-xunit/DesignTime/DesignTimeTestConverter.cs [hunk 1]
```diff
@@ -10,13 +10,15 @@ namespace Xunit.Runner.DotNet
...
     {
-#if DNXCORE50
+#if NETSTANDARDAPP1_5
         private readonly static HashAlgorithm Hash = SHA1.Create();
 #else
         private readonly static HashAlgorithm _hash = new SHA1Managed();
 #endif
         public static IDictionary<ITestCase, VsTestCase> Convert(IEnumerable<ITestCase> testcases)
         {
+            Console.WriteLine($"DEBUG: {testcases.Count()} test cases found.");
```
Comment (livarcocc) [inline src/dotnet-test-xunit/DesignTime/DesignTimeTestConverter.cs]: Will remove this. Left over.

File: src/dotnet-test-xunit/Program.cs [hunk 2]
```diff
@@ -405,8 +418,8 @@ private void SendTestCompletedIfNecessary(bool designTime, bool list)
             try
             {
                 // Turn off pre-enumeration of theories when we're not running in Visual Studio
-                if (!designTime)
-                    assembly.Configuration.PreEnumerateTheories = false;
+//                if (!designTime)
```
Comment (livarcocc) [inline src/dotnet-test-xunit/Program.cs]: And this.

**Comment by andyleejordan** [comment]
Yay! I welcome this happily.

**Comment by livarcocc** [comment]
@pranavkm With this change, I am hitting this exception when trying to run the tests. This only happens to our tests. Other tests using the same version of the runner are fine. I worked around it by simply keeping the tests in dnxcore50, but something is going on here.

Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'System, Version=2.0.5.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e, Retargetable=Yes' or one of its dependencies. The system cannot find the file specified.
   at Microsoft.Extensions.DependencyModel.DependencyContextJsonReader.Read(Stream stream)
   at Microsoft.Extensions.DependencyModel.DependencyContext.Load(Assembly assembly)
   at Xunit.Runner.DotNet.Program.GetAvailableRunnerReporters()
   at Xunit.Runner.DotNet.Program.Run(String[] args)
   at Xunit.Runner.DotNet.Program.Main(String[] args)

**Comment by TheRealPiotrP** [comment]
:shipit:

**Comment by pranavkm** [comment]
Are you still seeing the issue at runtime?

**Comment by livarcocc** [comment]
I did not try again. I switched the test TFM to dnxcore50 and imported netstandardapp1.5 and it worked.

**Pull request merged**